packaging/docker/alpine/Dockerfile

```diff
@@ -1,14 +1,11 @@
-FROM golang:alpine AS build
+FROM docker.io/library/golang:alpine AS build
 ENV CGO_ENABLED=0
-RUN apk add --no-cache --update git
+RUN apk add --no-cache --update git make
```

Comment:
Since we're refactoring stuff there, I think `--no-cache` and `--update` can be dropped.

Reply:
I have removed `--update` but kept `--no-cache` since that is still being recommended here: https://hub.docker.com/_/alpine